db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
```

Comment:
That's probably my fault. @baldowl can you discard the schema.rb changes from your PR?